src/main/resources/archetype-resources/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM payara/server-full
```

Comment:
This needs a version tag as soon the latest version will be jakarta namespace based.

Reply:
Done.